src/Microsoft.Health.Fhir.SqlServer/Features/Operations/Import/SqlImportOperation.cs

```diff
@@ -325,17 +364,56 @@ public async Task PostprocessAsync(CancellationToken cancellationToken)
             }
         }
 
-        private async Task<(string tableName, string indexName)> ExecuteRebuildIndexTaskAsync(string tableName, string indexName, bool pageCompression, CancellationToken cancellationToken)
+        private async Task<string> ExecuteSqlCommand(string tableName, string indexName, string command, CancellationToken cancellationToken)
+        {
+            _logger.LogInformation(string.Format("table: {0}, index: {1} rebuild index start at {2}", tableName, indexName, DateTime.Now.ToString("hh:mm:ss tt")));
+            try
+            {
+                using (SqlConnectionWrapper sqlConnectionWrapper = await _sqlConnectionWrapperFactory.ObtainSqlConnectionWrapperAsync(cancellationToken, true))
+                using (SqlCommandWrapper sqlCommandWrapper = sqlConnectionWrapper.CreateRetrySqlCommand())
+                {
+                    sqlCommandWrapper.CommandTimeout = _importTaskConfiguration.InfinitySqlLongRunningOperationTimeoutInSec;
+
+                    VLatest.ExecuteCommandForRebuildIndexes.PopulateCommand(sqlCommandWrapper, tableName, indexName, command);
+                    using SqlDataReader sqlDataReader = await sqlCommandWrapper.ExecuteReaderAsync(cancellationToken);
+                    while (await sqlDataReader.ReadAsync(cancellationToken))
+                    {
+                        indexName = sqlDataReader.GetString(0);
```

Comment:
What is the reason for reading out the index name and returning it? It looks like the method call above just ignores the returned value. Is this just for the logger message below?

Reply:
Yes, we use it for logger message.